src/assignment_list/assignmentlist.ts

```diff
@@ -293,6 +293,36 @@ class Assignment {
 
       }
     } else if (this.data.status == 'fetched') {
+        var refetchButton = document.createElement('button');
+        refetchButton.classList.add('btn', 'btn-danger', 'btn-xs');
+        refetchButton.setAttribute('data-bs-toggle', 'tooltip');
+        refetchButton.setAttribute('data-bs-placement', 'top');
+        refetchButton.setAttribute('style', 'background:#d43f3a; margin-left:5px');
```

Comment:
As said above, the style properties could be in *common.css*.

I would also suggest:
- an orange background
- the same font color as the neighbouring button
- slightly change the background color on hover